### PR TITLE
LPS-134472 Remove warnings from RoleFilter tests

### PR DESCRIPTION
**Pull request by ramonmsp**
Technical Debt: [LPS-129917](https://issues.liferay.com/browse/LPS-129917)

### CODE REVIEW

**Comment by ramonmsp** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/filter/AssigneeFilter.es.js [hunk 1]
```diff
@@ -36,23 +36,27 @@ const wrapper = ({children}) => (
 );
 
 describe('The assignee filter component should', () => {
-	afterEach(cleanup);
+	afterEach(() => cleanup());
```
Comment (naraandrade) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/filter/AssigneeFilter.es.js:39]: @ramonmsp The effect of this line isn't exact the same of the previous one?

**Comment by ramonmsp** [comment]
ci:test:sf

**Comment by ramonmsp** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**